conf/messages.en

```diff
@@ -239,14 +239,14 @@ invalidXMLFile.uploadLink = Upload a different file
 invalidXMLFile.print = Print this page
 
 fileCheck.title = We are checking your file
-fileCheck.heading = We are checking your file 
+fileCheck.p1 = We are checking your file
 
 sendYourFile.title = Send your file
 sendYourFile.heading = Send your file
-sendYourFile.p1 = We will now automatically check your file for business rule errors. This could take a few minutes.
+sendYourFile.p1 = We will now automatically check your file for record errors and send you to another page once completed. These checks could take a few minutes.
```

Comment:
@koladeadewuyi the ticket has been updated, this should now be "We will now automatically check your file for **business rule** errors and send you to another page once completed. These checks could take a few minutes."

Reply:
Good spot 👍 .
This is now fixed. Thanks